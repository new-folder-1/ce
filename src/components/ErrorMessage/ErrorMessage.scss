.ErrorMessage {
    color: #9f5252;
}
