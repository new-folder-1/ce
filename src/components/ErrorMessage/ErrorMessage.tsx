import React from 'react';

import './ErrorMessage.scss';

interface ErrorMessage {
    message: string;
}

export const ErrorMessage = ({ message }: ErrorMessage) => (
    <span className="ErrorMessage">
        <span>{'\u26A0'}</span>
        <span> {message}</span>
    </span>
);
