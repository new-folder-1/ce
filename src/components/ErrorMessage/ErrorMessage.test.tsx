import React from 'react';
import { shallow } from 'enzyme';

import { ErrorMessage } from './ErrorMessage';

describe('ErrorMessage', () => {
    it('snapshot', () => {
        expect(
            shallow(<ErrorMessage message="Failed to fetch wallets" />)
        ).toMatchSnapshot();
    });
});
