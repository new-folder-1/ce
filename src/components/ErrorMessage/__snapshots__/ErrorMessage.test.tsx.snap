// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage snapshot 1`] = `
<span
  className="ErrorMessage"
>
  <span>
    âš 
  </span>
  <span>
     
    Failed to fetch wallets
  </span>
</span>
`;
